src/ol/Tile.js

```diff
@@ -172,7 +172,11 @@ class Tile extends EventTarget {
    * @api
    */
   setState(state) {
-    if (this.state !== TileState.ERROR && this.state > state) {
+    if (
+      this.state !== TileState.ERROR &&
+      this.state !== TileState.EMPTY &&
```

Comment:
When a tile has an `EMPTY` state, we don't really care about the state sequence, because the change to `EMPTY` that happens before has cleared the listeners on the `TileQueue` already.

I added this commit because the previous "not strictly related change" made a test fail-